### DISCUSSION TITLE
Does sea-orm support pipelining?

### DISCUSSION THREAD

**Discussion by PreetamSing** [SeaQL/sea-orm#1510] [Q&A] [answered]
Here's what I mean by [pipelining](https://docs.rs/tokio-postgres/latest/tokio_postgres/index.html#pipelining).
I want to know whether a single connection in the pool is used for sending multiple requests simultaneously, handing the charge to database to decide if it can process those requests independently i.e. parallelly.

**Comment by tyt2y3** [comment, marked as answer]
No, as the underlying driver is SQLx, which does not support pipelining.

**Comment by PreetamSing** [comment]
For further reference to someone: https://github.com/launchbadge/sqlx/issues/408
